src/gen_dragonfruit/gd_board.cpp

```diff
@@ -116,7 +115,7 @@ static void gd_board_post(){
     uint16_t mpl115a2_press = gd_dev_adafruit_MPL115A2_press_read();
     Serial.print(F("[P] mpl115a2 pressure: "));
     Serial.print(mpl115a2_press);
-    Serial.print(".");
+    Serial.println(" Pa");
```

Comment:
This statement should be enclosed in the F() macro declaration to save SRAM space.